README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/mobile-shell/mosh.svg?branch=master)](https://travis-ci.org/mobile-shell/mosh)
```

Comment:
Can you update the change description to someting like "Replace Travis CI with Github Actions"?

Reply:
Done.